### PR TITLE
feat(common): env variable for config at project level

### PR DESCRIPTION
**Pull request by gbloquel**
Currently there is no way to manage gita config at project level.

For me a project is composed of a several repos.
My PR adds an environment variable `GITA_PROJECT_HOME`

### CODE REVIEW

**Review by nosarthur — APPROVED** [review]
lgtm

File: gita/common.py [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 
 def get_config_dir() -> str:
-    root = os.environ.get('XDG_CONFIG_HOME') or os.path.join(
-        os.path.expanduser('~'), '.config')
+    root = os.environ.get('GITA_PROJECT_HOME') or 
+           os.environ.get('XDG_CONFIG_HOME') or
```
Comment (nosarthur) [inline gita/common.py:6]: have u tried this on your computer? this will cause syntax error

Comment (gbloquel) [reply gita/common.py]: I fixed this syntax error.

**Comment by gbloquel** [comment]
Thank U for the merge.
But could you do a new release ?

**Comment by gbloquel** [comment]
Hi @nosarthur ,
Could you release and tag my contrib ?

**Pull request merged**